manifests/params.pp

```diff
@@ -1,3 +1,4 @@
+# Setting params for the class
```

Comment:
also, would we instead say that we're setting params for the module instead of the 'class'?